addons/l10n_pl/models/account_move.py

```diff
@@ -16,3 +17,9 @@ class AccountMove(models.Model):
         string='B_MPV_Prowizja',
         help="Supply of agency and other services pertaining to the transfer of a single-purpose voucher",
     )
+
+    def action_post(self):
+        "Validation to avoid having credit notes with more than the invoice"
+        if self.reversed_entry_id and self.reversed_entry_id.amount_total < self.amount_total:
```

Comment:
Since this is a limitation required for legal reasons in a specific country, I believe we should only apply that restriction for moves from the said country :thinking: 

```suggestion
        if self.country_code == "PL" and self.reversed_entry_id and self.reversed_entry_id.amount_total < self.amount_total:
```

Reply:
Indeed, I forgot that :')
Thanks for catching those!